cpg_workflows/stages/gcnv.py

```diff
@@ -82,8 +81,10 @@ def queue_jobs(self, seqgroup: SequencingGroup, inputs: StageInput) -> StageOutp
         if seqgroup.cram is None:
             raise ValueError(f'No CRAM file found for {seqgroup}')
 
+        assert seqgroup.dataset.cohort
+
         jobs = gcnv.collect_read_counts(
-            intervals_path=inputs.as_path(get_cohort(), PrepareIntervals, 'preprocessed'),
+            intervals_path=inputs.as_path(seqgroup.dataset.cohort, PrepareIntervals, 'preprocessed'),
```

Comment:
It would be good (for posterity) to briefly explain in a commit message why `get_cohort()` was the wrong choice for _gcnv.py_.